assets/js/buttons-examples.js

```diff
@@ -212,6 +212,20 @@ $(document).ready(function () {
           classes: "chat-btn",
           text: "",
         },
+        {
+          type: "close",
+          label: "Close Button",
+          classes: "",
+          variations: [
+             {  
+                 classes: "close-btn",
+                 label: "",
+                 text:""
+             }
+          ],
+          shouldHaveRoundedType: false,
```

Comment:
Please move properties after variation to before variation (for readability only)